post-install.yml

```diff
@@ -66,14 +66,22 @@
         tsuru-admin platform-add static -d https://raw.github.com/tsuru/basebuilder/master/static/Dockerfile
       when: "not 'static' in tsuru_platforms.stdout"
 
-    - name: Post Install Tasks | Tsuru key-list
+    - name: Post Install Tasks | Tsuru list tsuru_deployer key
       shell: >
-        tsuru key-list
+        tsuru key-list -n | awk '$2 == "tsuru_deployer" {print $5}'
```

Comment:
I think the `{print $5}` should be removed. It omits the key type (`ssh-rsa`) and causes the tasks to run every time:
```
TASK: [Post Install Tasks | Tsuru remove old key] *****************************
changed: [10.128.11.176]

TASK: [Post Install Tasks | Tsuru add ssh key] ********************************
changed: [10.128.11.176]
```